tck/package.json

```diff
@@ -11,7 +11,8 @@
   },
   "devDependencies": {
     "testcontainers": "7.10.0",
-    "typescript": "4.3.4"
+    "typescript": "4.3.4",
+    "protobufjs": "6.11.3"
```

Comment:
This was to work around https://github.com/lightbend/kalix-javascript-sdk/issues/398 until we have a better solution.